src/xenia/gpu/gl4/command_processor.cc

```diff
@@ -2187,8 +2186,9 @@ CommandProcessor::UpdateStatus CommandProcessor::UpdateDepthStencilState() {
 
   bool dirty = false;
   dirty |= SetShadowRegister(regs.rb_depthcontrol, XE_GPU_REG_RB_DEPTHCONTROL);
-  dirty |=
-      SetShadowRegister(regs.rb_stencilrefmask, XE_GPU_REG_RB_STENCILREFMASK);
+  dirty |= SetShadowRegister(regs.rb_stencilrefmask, XE_GPU_REG_RB_STENCILREFMASK);
+  dirty |= SetShadowRegister(regs.rb_stencilrefmaskbf, XE_GPU_REG_RB_STENCILREFMASK_BF);
```

Comment:
Probably i need to check out games that have two sided enabled so as to confirm this functionality.